src/about-us/aboutUs.js

```diff
@@ -0,0 +1,32 @@
+import './aboutUsStyles.css';
+import aboutUsImage from '../images/about-us.jpg';
+
+const AboutUsPage = (() => {
+  const createAboutUsPageMarkUp = () => `
+    <div id="aboutUsId" class="about-us page d-none">
+    <article class="about-us__page">
+      <img id="about-us-image"/>
+      
+      <div class="about-us__info">
+        <h2 class="about-us__title">About us</h2>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus aliquid qui sint placeat tempora aperiam ducimus, odit cum omnis expedita minus accusantium! Debitis fugiat numquam officia eveniet nostrum ut et.</p>
+      </div>
+    </article>
+  </div>
```

Comment:
We need to use `document.createElement` and `document.appendChild` to appends HTML elements to the `<div>` with `id=content`

![image](https://user-images.githubusercontent.com/31897434/89599791-67e4f000-d858-11ea-97fe-976b47c8b29c.png)

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page